frontend/src/types.ts

```diff
@@ -883,7 +883,7 @@ export interface FilterType {
     interval?: IntervalType
     date_from?: string | null
     date_to?: string | null
-    properties?: PropertyFilter[]
+    properties?: PropertyGroupFilter
```

Comment:
@EDsCODE @neilkakkar Before I refactor a bunch of stuff, I'm confirming this is the change we're looking for? We will only handle that format for properties moving forwards?

Reply:
the API can accept either. We still need to be backwards compatible with the properties that exist in saved insights so I dont think a complete type swapover will work. hmmm

Reply:
yeah that's true 😬